### PR TITLE
feat(footer-harmonised): Review markup and add controls - FRONT-1845

### PR DESCRIPTION
**Pull request by papegaill**

### CODE REVIEW

File: src/implementations/twig/components/footer-harmonised/eu-footer-harmonised.story.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    res.rows[0].splice(2, 1);
+  }
+  if (!args.hide_contact && !args.hide_follow) {
+    res.rows[0].splice(1, 1);
+  }
+
+  return res;
+};
+
+export const Default = (args) => footer(prepareDataG1(specsEu, args));
```
Comment (emeryro) [inline src/implementations/twig/components/footer-harmonised/eu-footer-harmonised.story.js:75]: do we really need to have this whole file, if we don't display EU harmonized footer at all?

File: src/implementations/vanilla/components/footer-harmonised/footer-harmonised-eu.scss [hunk 2]
```diff
@@ -0,0 +1,176 @@
+/**
+ * Footer Harmonised component
```
Comment (emeryro) [inline src/implementations/vanilla/components/footer-harmonised/footer-harmonised-eu.scss:2]: same question here: is this css actually used somewhere?

File: src/specs/components/footer-harmonised/demo/data--eu.js [hunk 3]
```diff
@@ -0,0 +1,349 @@
+module.exports = {
+  rows: [
```
Comment (emeryro) [inline src/specs/components/footer-harmonised/demo/data--eu.js:2]: same here :)

File: src/implementations/twig/components/footer-harmonised/footer-harmonised.html.twig [hunk 4]
```diff
@@ -1,21 +1,19 @@
 {% spaceless %}
 
-{#
+{# 
   Parameters:
   - "group" (string) (default: 'group1'): 'group1', 'group2' or 'group3'
-  - "sections" (array of objects) (default: []):
+  - "rows" (array of objects) (default: [])
```
Comment (emeryro) [inline src/implementations/twig/components/footer-harmonised/footer-harmonised.html.twig:6]: we should update the readme file of the component too

**Review by emeryro — CHANGES_REQUESTED** [review]
thanks for this!
I spotted a small design difference on EC harmonized group 1:
before:
image

after:
image

File: src/implementations/twig/components/footer-harmonised/eu-footer-harmonised.story.js (hunk 1 above)
Comment (papegaill) [reply src/implementations/twig/components/footer-harmonised/eu-footer-harmonised.story.js:75]: Thanks @emeryro
The story for EU has a different name from the one of EC 
if you use the EC | EU switcher, then yes you end up with a blank page.

EU: https://ecl-preview-1938--europa-component-library.netlify.app/playground/eu/?path=/story/components-footers-core--default

**Review by emeryro — COMMENTED** [review]
that's good to me. Could you just update the tests snapshots? Then we can merge it

**Review by emeryro — APPROVED** [review]

**Pull request merged**